tests/0113-cooperative_rebalance.cpp

```diff
@@ -2073,11 +2091,10 @@ static void poll_all_consumers (RdKafka::KafkaConsumer **consumers,
  *
  * @param subscription_variation 0..2
  *
- * FIXME: What's the stressy part?
  * TODO: incorporate committing offsets.
  */
 
-static void u_stress (bool use_rebalance_cb, int subscription_variation) {
+static void u_complex (bool use_rebalance_cb, int subscription_variation) {
```

Comment:
Still a subjective value word that doesn't really reflect what is going on ;)
u_multiple_subscription_changes()